CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## Unreleased
-- None
+- [#106](https://github.com/Studiosity/grover/pull/106) Add support for addStyleTag and addScriptTag
```

Comment:
Don't forget to reference yourself. See L16 for an example. Then make sure you add yourself at the bottom of the file ;)